race_test.go

```diff
@@ -15,8 +15,6 @@
  * limitations under the License.
  */
 
-// +build race
-
```

Comment:
Curious what this change is about? If it is intentional, perhaps the test should be moved to another file now.

Reply:
This is from the first commit of the PR. With Go 1.13, this was causing the following build error:
```
./race_test.go:18:1: +build comment must appear before package clause and be followed by a blank line
```

This was due to some bug in go vet that is fixed in Go 1.14. It has something to do with the block comment used for the license. It's not worth breaking Go 1.13 builds to disable this test for non-race builds.

Reply:
Ah, didn't read the commit message. Makes sense.

Reply:
I just stumbled on the reason why this wasn't working while catching up on the latest collection of go design drafts: https://youtu.be/AgR_mdC4Rs4?t=488. I guess go vet was correctly complaining. I wonder why it went quiet on 1.14.